package.json

```diff
@@ -37,6 +37,7 @@
     "pretest": "npm run prepare && npm run compile"
   },
   "dependencies": {
+    "@google-cloud/promisify": "^0.3.1",
```

Comment:
You don't need this :) The `util` module is baked into nodejs, and `util.promisify` works just fine.

Reply:
@JustinBeckwith I tried that, but it's failing in node-6 build, for .ts